### DISCUSSION TITLE
Multiple Language support

### DISCUSSION THREAD

**Discussion by sirKitKat** [sct/overseerr#531] [Feature Requests]
My collection of films in Plex are divided in 2 libraries: English and Dutch
Also, Radarr has 2 profiles for Dutch and English films.

It would be nice in addition of the HD and 4K quality profiles to support different language profiles in overseerr

**Comment by sct** [comment]
Is this what you are asking for? https://github.com/sct/overseerr/issues/366

**Reply by sirKitKat** [reply]
Looks like it :) thanks if this is already on the roadmap 👍

**Reply by insingh50** [reply]
It seems that’s not requesting the same thing. The linked ticket seems to be for Sonarr. OP was asking about Radarr. 
I think what is being requested is that when a movie is being requested, the default profile selected should be chosen based on the language of the movie.  At least that’s the feature that I’d like to request if OP is not.

**Reply by TheCatLady** [reply]
@insingh50 You can simply select `Original` for the profile language in Radarr.

**Comment by behead974** [comment]
Bonjour, est-il possible de mettre les notifications en français ??
